protocol/messenger.go

```diff
@@ -4164,6 +4165,35 @@ func (m *Messenger) prepareMessage(msg *common.Message, s *server.MediaServer) {
 	if msg.ContentType == protobuf.ChatMessage_IMAGE {
 		msg.ImageLocalURL = s.MakeImageURL(msg.ID)
 	}
+	if msg.ContentType == protobuf.ChatMessage_DISCORD_MESSAGE {
+
+		dm := msg.GetDiscordMessage()
+		exists, err := m.persistence.HasDiscordMessageAuthorImagePayload(dm.Author.Id)
+		if err != nil {
+			return
+		}
+
+		if exists {
+			dm.Author.LocalUrl = s.MakeDiscordAuthorAvatarURL(dm.Author.Id)
+		}
+
+		for idx, attachment := range dm.Attachments {
+			if strings.Contains(attachment.ContentType, "image") {
+				hasPayload, err := m.persistence.HasDiscordMessageAttachmentPayload(attachment.Id)
+				if err != nil {
+					m.logger.Error("failed to check if message attachment exist", zap.Error(err))
+					continue
+				}
+				if hasPayload {
+					localURL := s.MakeDiscordAttachmentURL(dm.Id, attachment.Id)
+					dm.Attachments[idx].LocalUrl = localURL
+				}
+			}
```

Comment:
If the attachment is a type of image do something, else ignore the attachment.

I presume in the future this can be added to